modules/aws-codebuild-step/example/main.tf

```diff
@@ -0,0 +1,178 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "build" {
+  source = "../"
+
+  project_name    = "build-${var.environment}"
+  description     = "thing_that_build"
+  environment     = var.environment
+  pipeline_s3_arn = aws_s3_bucket.codepipeline_bucket.arn
+
+  buildspec_path = ".aws/dev/build.yaml"
```

Comment:
I don't get it. What is this file? Where is it?

Reply:
in artifacts from previous step (eg. github)
travis uses "/.travis.ci" 
on codebuild you provide path that is going to be read to get commands to run